frontend/app/src/lib/breadcrumbs.ts

```diff
@@ -1,4 +1,37 @@
-import { TenantedPath } from '@/router';
+export type TenantedPath =
```

Comment:
we should probably consider doing something else with this component

Reply:
I think we might not even need this anymore - it was mostly to keep track of things for the V0/1 split, but I wonder if there's a simpler way here